io.catenax.material_demand/1.0.0/gen/MaterialDemand.html

```diff
@@ -0,0 +1,1889 @@
+
+<!doctype html>
```

Comment:
You can remove completely the 'gen' folder as this will be generated at run time automatically.

Reply:
folder removed